### PR TITLE
New: Conversion Center - Contact Buttons v4 - Final elements

### PR DESCRIPTION
**Pull request by mserino**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Adding final tweaks to Contact Buttons v4

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss [hunk 1]
```diff
@@ -84,41 +85,80 @@
...
+			border-width: 8px 0 8px 10px;
+			content: "";
+			height: 0;
+			position: absolute;
+			inset-inline-end: -8px;
+			top: 50%;
+			transform: translateY(-50%);
+			width: 0;
+			
+			@if $direction == rtl {
```
Comment (hein-obox) [inline modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss:158]: If possible, please replace with logical properties, such as `border-inline-start-width`. 
I belief that that is fully supported by modern browsers, see https://caniuse.com/?search=border-inline-start-width.

File: modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss [hunk 2]
```diff
@@ -84,41 +85,80 @@
...
+		color: var(--e-contact-buttons-tooltip-text);
+		font-size: 13px;
+		line-height: 25px;
+		padding: 4px 14px;
+		position: relative;
+
+		&::after {
+			border-color: transparent transparent transparent var(--e-contact-buttons-tooltip-bg);
+			border-style: solid;
+			border-width: 8px 0 8px 10px;
```
Comment (hein-obox) [inline modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss:149]: Maybe replace with logical properties without the shorthand, like: 
border-inline-start-width
border-inline-end-width
border-block-start-width
border-block-end-width

File: modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss (hunk 1 above)
Comment (mserino) [reply modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss]: Hi @hein-obox thanks for the suggestion, great stuff!
I've also replaced another border that was using the same logic, please see here https://github.com/elementor/elementor/pull/27312/commits/0a5e770e2ed5a8d6a4bddb09c58855f5e4badf7b#diff-37301fb7f13b4f650234f35baf9163eb32daf4964e29c32a0b189f0f927e1537R393

File: modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss (hunk 2 above)
Comment (mserino) [reply modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss]: Thanks!

**Review by hein-obox — APPROVED** [review]

**Pull request merged**